pkg/llmvalidate/llmvalidate_test.go

```diff
@@ -0,0 +1,36 @@
+package llmvalidate
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetPromptsForCodepathWithSubPaths(t *testing.T) {
+	codePath := filepath.Join("testdata", "testplugin")
+
+	prompts, err := getPromptContentForCode(codePath, []string{"src", "pkg"})
+
+	require.NoError(t, err)
+	require.Len(t, prompts, 2)
+}
+
+func TestGetPromptsForCodepathWithoutSubPaths(t *testing.T) {
+	codePath := filepath.Join("testdata", "testplugin")
+
+	prompts, err := getPromptContentForCode(codePath, []string{"."})
```

Comment:
Nit: Maybe a test for `prompts, err := getPromptContentForCode(codePath, []string{})` since it will automatically set this if empty right?

Reply:
Good idea. let me add it.